src/environmentd/tests/server.rs

```diff
@@ -2676,6 +2676,28 @@ fn test_webhook_duplicate_headers() {
     assert_eq!(resp.status().as_u16(), 401);
 }
 
+#[mz_ore::test]
+#[cfg_attr(miri, ignore)] // too slow
+fn test_webhook_invalid_path() {
+    let server = test_util::start_server(test_util::Config::default()).unwrap();
+    let http_client = Client::new();
+
+    let webhook_url = format!(
+        "http://{}/api/webhook/materialize/public/webhook_text/extra_component",
+        server.inner.http_local_addr()
+    );
+
+    // Send a request with duplicate headers.
```

Comment:
I don't understand this comment.

Reply:
Argh, sorry this was a copy-paste failure